AmazonChimeSDK/AmazonChimeSDK/internal/audio/protocols/AudioClientProtocol.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AudioClientInterface.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import AmazonChimeSDKMedia
+import Foundation
+
+@objc public protocol AudioClientProtocol {
+    func startSession(_ host: String!, basePort port: Int, callId: String!, profileId: String!, microphoneMute mic_mute: Bool, speakerMute spk_mute: Bool, isPresenter presenter: Bool, sessionToken tokenString: String!, audioWsUrl: String!, callKitEnabled: Bool) -> audio_client_status_t
```

Comment:
Can we make these parameters optional instead of force unwrapping them? `String?` instead of `String!`

Reply:
The func signature is generated from ObjC definitions of the header file exposed. It's possible to make these params optional on swift side but we may need to add nullable annotations in ObjC: https://developer.apple.com/swift/blog/?id=25, otherwise there will be compiler warnings because it would be changing the method signature.

Reply:
Discussed offline, since we do not want to generate any warnings and we cannot just add nullability annotation to a single API without Xcode complaining about all other APIs, this needs to stay as is.